### DISCUSSION TITLE
Why kube-prometheus used metrics.k8s.io instand of custom.metrics.k8s.io?

### DISCUSSION THREAD

**Discussion by terloo** [prometheus-operator/kube-prometheus#1904] [Q&A]
As I know metrics api group maybe distributed like this

|            | API Group               | application                                         
| ---------------- | ----------------------- | ------------------------------------------------- | 
| Resource Metrics | metrics.k8s.io          | metrics-server                                   |
| Custom Metrics   | custom.metrics.k8s.io   |PrometheusAdaptor |
| External Metrics | external.metrics.k8s.io | Cloud Provider                                |

After I deploy the mainifest, I found promethes-adaptor used apiGroup metrics.k8s.io in kube-prometheus.But use custom.metrics.k8s.io in his self project.
Why is this?

**Comment by metalmatze** [comment]
This is probably best asked on https://github.com/kubernetes-sigs/prometheus-adapter directly.

**Reply by terloo** [reply]
That's why I'm confused.
Prometheus-adapter actually use group custom.metrics.k8s.io  in  [custom-metrics-apiservice.yaml](https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/deploy/manifests/custom-metrics-apiservice.yaml)
